src/components/main/Result.js

```diff
@@ -73,7 +73,7 @@ class Result extends React.Component {
 
         return (
             <div id="results">
-                <h1 className={ calc <0 ?"negCalc" : ""}>{calc.toFixed(0)}</h1>
+                <h1 class={ calc <0 ?"negCalc" : ""}>{calc.toFixed(0)}</h1>
```

Comment:
This has to be className, not class. During our last meeting, I saw the error come up in the console and fixed this so not sure why this got changed back. "class" is not recognized in ReactJS. Please change it back to "className".